qiita_pet/handlers/qiita_redbiom.py

```diff
@@ -34,14 +34,14 @@ def _redbiom_metadata_search(self, query, contexts):
         query = query.lower()
         try:
             samples = redbiom.search.metadata_full(query, False)
-        except TypeError:
-            message = (
-                'Not a valid search: "%s", are you sure this is a '
-                'valid metadata value?' % query)
         except ValueError:
             message = (
                 'Not a valid search: "%s", your query is too small '
                 '(too few letters), try a longer query' % query)
+        except:
+            message = (
+                'Not a valid search: "%s", are you sure this is a '
+                'valid query value?' % query)
```

Comment:
They probably won't be sure. What about: "The query ("%s") did not work and may be malformed. Please check the search help for more information on the queries."